Procfile

```diff
@@ -0,0 +1,2 @@
+web: java -Dserver.port=$PORT  -jar target/*.jar --spring.datasource.url=$SPRING_DATASOURCE_URL --spring.datasource.username=$SPRING_DATASOURCE_USERNAME --spring.datasource.password=$SPRING_DATASOURCE_PASSWORD
```

Comment:
now it's excessive since you deploy through docker?

Reply:
when the deployment did not work for me, I read about the need to leave it. noe everythink is work. And I removed this one